.azure-pipelines/dependency-check.yml

```diff
@@ -4,8 +4,8 @@ steps:
 
     pip3 install natsort
 
-    CHECK_RESULT=$(python3 ./.azure-pipelines/dependency_check/dependency_check.py tests)
-    if [[ "$CHECK_RESULT" == "True" ]]; then
+    python3 ./.azure-pipelines/dependency_check/dependency_check.py tests
+    if [[ $? -eq 1 ]]; then
```

Comment:
Why only rc=1 is considered as failure? What about other rc values that may be added in the future?
Is this format more robust and extensible? `if [[ $? -neq 0 ]]; then`?

Reply:
modified